guides/v2.3/howdoi/checkout/checkout-add-custom-carrier.md

```diff
@@ -0,0 +1,299 @@
+---
```

Comment:
See comments for v2.2 version.

Reply:
I've implemented all notices to 2.2 version.

Reply:
I'm finding that you didn't complete suggested revisions in v2.1 and didn't apply those same changes to v2.2 and v2.3. Please review and fix in each version.

Reply:
@shrielenee , I've added missed suggestions. Could you please review?

Many thanks!!!